windows/vhba_hot_add_remove/vhba_test.yml

```diff
@@ -6,6 +6,32 @@
 # - lsilogic: no inbox driver in Windows guest OS now, so no testing.
 # - buslogic: not supported in 64bit Windows guest OS, so no testing.
 #
+- name: "Set fact of whether VM NVMe version 1.3 is enabled by default"
+  ansible.builtin.set_fact:
+    vm_nvme_spec13_enabled: >-
+      {{
+         vm_hardware_version_num | int >= 21 and 
+         ((guest_os_product_type == 'server' and guest_os_build_num | int >= 20348) or 
+         (guest_os_product_type == 'client' and guest_os_build_num | int >= 22000))
```

Comment:
Is the feature enabled for guest id or guest OS release?

Reply:
It's guest ID in a high probability, so changed to use guest ID here. Thanks.